source/install/ee-install.rst

```diff
@@ -4,43 +4,46 @@
 Enterprise Install and Upgrade
 ===========================================
 
-Mattermost Enterprise Edition is free to use in "team mode" without enterprise features enabled. Enable enterprise features using a `trial license <https://about.mattermost.com/trial/>`__ or by `purchasing a license key <https://about.mattermost.com/pricing/>`__. There are two variants of Enterprise Edition, E10 and E20, and you can `compare their features online <https://mattermost.com/pricing-feature-comparison/>`__.
+Mattermost Enterprise Edition is free to use in "team mode" without enterprise features enabled. There are two variants of Enterprise Edition, E10 and E20, and you can `compare their features online <https://mattermost.com/pricing-feature-comparison/>`__.
+
+Once you've downloaded and installed Mattermost, start an Enterprise E20 trial via **Main Menu > System Console > Edition and License > Start trial**. You can view `pricing and feature information <https://mattermost.com/pricing/>`__ and purchase a license key online <https://customers.mattermost.com/login/>`__.
```

Comment:
The link at the end (to customers.mattermost.com/login) is messed up, I think it's missing a ` character or something

Reply:
Thanks for catching that! Sorted.